staticMessage: Setter for styleClass missing

[StaticMessageBase](https://github.com/primefaces/primefaces/blob/ba2900cfcd4276e31224f38667561973b36a7ba9/src/main/java/org/primefaces/component/staticmessage/StaticMessageBase.java#L96) seems to just miss a valid setter for styleClass.

## 1) Environment
- PrimeFaces version: 7.0
- Does it work on the newest released PrimeFaces version? Version? - No
- Does it work on the newest sources in GitHub? - Don't think so as there is still no valid setter
- Application server + version: JBoss 7.1
- Affected browsers: Chrome 75.0.3770.100 64 Bit

## 2) Expected behavior
When setting a `styleClass` for a `p:staticMessage` the styleClass should be used by the component.

## 3) Actual behavior
An exception is thrown as there is no valid setter for styleClass:
```
Caused by: java.lang.IllegalArgumentException: Setter not found for property styleClass
	at javax.faces.component.UIComponentBase$AttributesMap.put(UIComponentBase.java:2468)
	at javax.faces.component.UIComponentBase$AttributesMap.put(UIComponentBase.java:2336)
	at com.sun.faces.facelets.tag.jsf.ComponentRule$LiteralAttributeMetadata.applyMetadata(ComponentRule.java:87)
	at com.sun.faces.facelets.tag.MetadataImpl.applyMetadata(MetadataImpl.java:81)
	at javax.faces.view.facelets.MetaTagHandler.setAttributes(MetaTagHandler.java:129)
	at javax.faces.view.facelets.DelegatingMetaTagHandler.setAttributes(DelegatingMetaTagHandler.java:102)
```

## 4) Steps to reproduce
This can stable be reproduced by execution the following XHTML.

## 5) Sample XHTML
```
<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://primefaces.org/ui">
   <body>
      <p:staticMessage detail="Hello there!" severity="warn" styleClass="large"/>
   </body>
</html>
```

## 6) Sample bean
not necessary

--- a/src/main/java/org/primefaces/component/staticmessage/StaticMessageBase.java
+++ b/src/main/java/org/primefaces/component/staticmessage/StaticMessageBase.java
@@ -93,7 +93,7 @@ public abstract class StaticMessageBase extends UIComponentBase {
         return (String) getStateHelper().eval(PropertyKeys.styleClass, null);
     }
 
-    public void getStyleClass(String styleClass) {
+    public void setStyleClass(String styleClass) {
         getStateHelper().put(PropertyKeys.styleClass, styleClass);
     }
 }

